src/Text/Libyaml.hs

```diff
@@ -77,6 +79,12 @@ data Style = Any
            | PlainNoTag
     deriving (Show, Read, Eq, Enum, Bounded, Ord, Data, Typeable)
 
+data SequenceStyle = AnySequence | BlockSequence | FlowSequence
+    deriving (Show, Eq, Enum, Bounded, Ord, Data, Typeable)
+
+data MappingStyle = AnyMapping | BlockMapping | FlowMapping
```

Comment:
What's the semantics for `AnyMapping`?

Reply:
See slightly fuller note below. I understand AnyMapping / AnySequence express no preference as to style to emit (block being in most cases the default). The parser should always tell you block or flow.